README.md

```diff
@@ -75,12 +75,16 @@ a suite file defines a series of tests to run.
 #### suite file format
 
 ```ruby
+# format description:
 # each test can be executed by either target duration using :time => N secs
 # or by number of events with :events => N
 #
+# you can specify the number of filter worker threads for each test with :workers => N
+# if you don't specify workers, it defaults to 1
```

Comment:
we should add a note here about version 2.0 improvement. Since 2.0 is released and GA, workers will be assigned to half the number of cpus, but for the case of benchmarks I like the default 1 as it makes things easier to understand. What do you think?